### DISCUSSION TITLE
Spec files: Execution issue

### DISCUSSION THREAD

**Discussion by bajipogadadanda** [webdriverio/codemod#103] [General]
Hi,
I have a test suite to execute all the spec files( fullReg:['test/specs/**/*.js']).
When I executed it first time, only 20% Spec has been executed, no Pass/Fail/Skip status for remaining Specs/Tests.
I increased the 'maxInstances' to 4 for capabilities(its chrome browser) then re-tested,  55% Specs has been executed.
So, I would like to understand how bulk Spec execution can be handled in WDIO? why other spec files were not executed by WDIO? why no status for pending Specs(even Skipped status) .Pending spec shd have some status and reason for not being executed.

**Comment by christian-bromann** [comment]
> why other spec files were not executed by WDIO?

Maybe a configuration issue? WebdriverIO should execute all files that match given glob pattern.

> why no status for pending Specs(even Skipped status) .Pending spec shd have some status and reason for not being executed.

What did WebdriverIO return?

**Reply by bajipogadadanda** [reply]
Hi,
No status, find the latest execution summary.
No status and reason why only 52% has been covered. In-fact I tried it with max of 7 instances(21 spec files and 7 instances).

2023-05-24T17:20:20.994Z DEBUG @wdio/cli:utils: Finished to run "onComplete" hook in 3ms

Spec Files:      7 passed, 4 failed, 21 total (52% completed) in 00:05:52

**Reply by bajipogadadanda** [reply]
Hi Chris,
Let me know if this to be turned into an issue or any procedural issue?
Still seeing the same problem.

**Reply by christian-bromann** [reply]
Sorry @Bapugit I don't have much time to look into these things. If you see any bugs we appreciate any contributions.
